src/layouts/layout.astro

```diff
@@ -112,14 +112,14 @@ import package_ from "../../package.json";
 			<ul>
 				<li>
 					Mastodon (<span data-translate="incl">incl.</span>&#x0020;<span
-						data-enumerate="Hometown,Fedibird,GlitchCafé"
-						>Hometown, Fedibird, GlitchCafé</span
+						data-enumerate="Hometown,Fedibird,GlitchCafé,Chuckya"
+						>Hometown, Fedibird, GlitchCafé, Chuckya</span
 					>)
 				</li>
 				<li>
 					Misskey (<span data-translate="incl">incl.</span>&#x0020;<span
-						data-enumerate="Firefish,Calckey,FoundKey,Meisskey"
-						>Firefish, Calckey, FoundKey, Meisskey</span
+						data-enumerate="Firefish(ex. Calckey),FoundKey,Misskey,Sharkey,Iceshrimp"
```

Comment:
**issue:** Did you verify that those actually work?

Adding a new project is more than just adding its name; even if it's just a fork.

Reply:
yeah